pulsar/managers/util/job_script/DEFAULT_JOB_FILE_TEMPLATE.sh

```diff
@@ -4,7 +4,8 @@ $headers
 
 _galaxy_setup_environment() {
     local _use_framework_galaxy="$1"
-    _GALAXY_JOB_HOME_DIR="$working_directory/../home"
+    _GALAXY_JOB_DIR="$working_directory"
+    _GALAXY_JOB_HOME_DIR="$working_directory/home"
```

Comment:
But then home will then be right in the working directory. We don't do that in Galaxy and might impact recursively collecting outputs, for instance.